pkg/artifacts/signable.go

```diff
@@ -163,7 +162,7 @@ func (oa *OCIArtifact) ExtractObjects(ctx context.Context, obj objects.TektonObj
 		imageResourceNames := map[string]*image{}
 		if tr.Status.TaskSpec != nil && tr.Status.TaskSpec.Resources != nil {
 			for _, output := range tr.Status.TaskSpec.Resources.Outputs {
-				if output.Type == v1alpha1.PipelineResourceTypeImage {
```

Comment:
😢 Pipelines probably shouldn't have removed these.

But if we have to back-port it let's break these into our own consts (perhaps under `internal/backport`?)

Reply:
Good call! Done.